modules/storage/helper.go

```diff
@@ -61,3 +65,34 @@ func toConfig(exemplar, cfg interface{}) (interface{}, error) {
 	}
 	return newVal.Elem().Interface(), nil
 }
+
+var (
+	_                    ObjectStorage = discardStorage("")
```

Comment:
Is this needed?

Reply:
Just make sure the `discardStorage` implements `ObjectStorage`. But maybe you are right, if the implementation has been broken, we will find it easily even without this line.

Reply:
The line below does the same or not?